recipes/pyemojify/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pyemojify" %}
+{% set version = "0.2.0" %}
+{% set md5sum = "5df175192fe507e6f8e13f12cf6dea3b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5sum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
If you aren't using [selectors](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors) you can use [noarch](https://conda-forge.org/docs/meta.html#building-noarch-packages).